specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/gallery.json

```diff
@@ -2567,7 +2578,7 @@
             "Premium_LRS"
           ],
           "x-ms-enum": {
-            "name": "StorageAccountType",
+            "name": "EdgeZoneStorageAccountType",
```

Comment:
Changing an enum name is a pretty significant breaking change, one that is not allowed in a new API Version release. 
This will need a breaking change request, https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes .

1. How many users use this enum and would be broken by this change? 
2. Was this enum nonfunctional before this change? 
3. Why is this name being changed?

Reply:
fixed